stripe-ui-core/src/main/java/com/stripe/android/uicore/elements/PhoneNumberController.kt

```diff
@@ -155,12 +156,14 @@ class PhoneNumberController(
                     initialPhoneNumber = e164Number.removePrefix(prefix),
                     initiallySelectedCountryCode = formatter.countryCode,
                     showOptionalLabel = showOptionalLabel,
+                    acceptAnyInput = acceptAnyInput
                 )
             } else {
                 PhoneNumberController(
```

Comment:
Can we make the constructor private to prevent this issue from occurring again?

Reply:
yep! Updated the PR so that the constructor is private now